[FR] Update `MWC/2017_4K` (#7657)
ppy/osu-wiki: wiki/Tournaments/MWC/2017_4K/fr.md

---
tags:
  - MWC 2017
  - MWC 4K 2017
  - MWC2017
  - MWC4K 2017
---

# osu!mania 4K World Cup 2017

> ![MWC 4K 2017 Logo](img/logo.jpg)

Reviewer: ```suggestion
![Logo de la MWC 4K 2017](img/logo.jpg)
```

> La **osu!mania 4K World Cup 2017** (***MWC 4K 2017***) est un tournoi d'osu!mania entre pays organisé par [l'équipe osu!](/wiki/People/The_Team). Il s'agit de la quatrième édition de la osu!mania World Cup.

Reviewer: ```suggestion
L'**osu!mania 4K World Cup 2017** (***MWC 4K 2017***) est un tournoi d'osu!mania 4 par pays organisé par [l'équipe osu!](/wiki/People/The_Team). Il s'agit de la quatrième édition de l'osu!mania 4K World Cup.
```

## Calendrier du tournoi

| Évènement | Date |
| --: | :-- |
| Inscription des équipes | 10/07/2017 - 23/07/2017 |
| Tirages au sort | 07/08/2017 (14:00 UTC) |
| Phases de poules | 12/08/2017 - 13/08/2017 |
| Huitièmes de finale | 20/08/2017 |
| Quarts de finale | 26/08/2017 - 27/08/2017 |
| Demi-finales | 02/09/2017 - 03/09/2017 |
| Finales - Semaine 1 | 09/09/2017 - 10/09/2017 |
| Finales - Semaine 2 | 16/09/2017 |

## Récompenses

| Place | Récompenses |
| :-- | :-- |

> | ![Couronne en or](/wiki/shared/crown-gold.png "1ère place") | 150 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil, et le titre « osu!mania Champion » pendant un an |

Reviewer: ```suggestion
| ![Couronne en or](/wiki/shared/crown-gold.png "1ère place") | 150 $ par membre de l'équipe, un badge de profil unique, de la marchandise exclusive d'osu!, le titre "osu!mania Champion" pendant un an |
```

> | ![Couronne en argent](/wiki/shared/crown-silver.png "2ème place") | 80 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil |

Reviewer: ```suggestion
| ![Couronne d'argent](/wiki/shared/crown-silver.png "2ème place") | 80 $ par membre de l'équipe, un badge de profil unique, de la marchandise exclusive d'osu! |
```

> | ![Couronne en bronze](/wiki/shared/crown-bronze.png "3ème place") | 40 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil |

Reviewer: ```suggestion
| ![Couronne de bronze](/wiki/shared/crown-bronze.png "3ème place") | 40 $ par membre de l'équipe, un badge de profil unique, de la marchandise exclusive d'osu! |
```

> ![](img/badge.jpg "MWC 4K 2017 winner badge") ![](img/badge2.jpg "MWC 4K 2017 2nd place badge") ![](img/badge3.jpg "MWC 4K 2017 3rd place badge")

Reviewer: ```suggestion
![](img/badge.jpg "Badge des gagnants de la MWC 4K 2017") ![](img/badge2.jpg "Badge de la 2ème place de la MWC 4K 2017") ![](img/badge3.jpg "Badge de la 3ème place de la MWC 4K 2017")
```

## Organisation

> L'osu!mania 4K World Cup 2017 était organisée par plusieurs membres de la communauté

Reviewer: ```suggestion
L'osu!mania 4K World Cup 2017 était organisée par divers membres de la communauté.
```

---
tags:
  - MWC 2017
  - MWC 4K 2017
  - MWC2017
  - MWC4K 2017
---

# osu!mania 4K World Cup 2017

![Logo de la MWC 4K 2017](img/logo.jpg)

> L'**osu!mania 4K World Cup 2017** (***MWC 4K 2017***) est un tournoi d'osu!mania 4 par pays organisé par [l'équipe osu!](/wiki/People/The_Team). Il s'agit de la quatrième édition de l'osu!mania 4K World Cup.

Reviewer: ```suggestion
L'**osu!mania 4K World Cup 2017** (***MWC 4K 2017***) est un tournoi d'osu!mania 4K par pays organisé par [l'équipe osu!](/wiki/People/The_Team). Il s'agit de la quatrième édition de l'osu!mania 4K World Cup.
```

> - [Topic de discussion](https://osu.ppy.sh/community/forums/topics/615787)

Reviewer: ```suggestion
- [Fil de discussion](https://osu.ppy.sh/community/forums/topics/615787)
```

- [Diffusion en direct](https://www.twitch.tv/osulive)

> - **[Statistiques générales, groupes et plus!](https://docs.google.com/spreadsheets/d/1qAbKpW3yCJ5U2qUoUoIob6qsmNW08LenxeL0FptdjxA/pubhtml)**

Reviewer: ```suggestion
- **[Statistiques générales, groupes et plus !](https://docs.google.com/spreadsheets/d/1qAbKpW3yCJ5U2qUoUoIob6qsmNW08LenxeL0FptdjxA/pubhtml)**
```

| ::{ flag=US }:: | **États-Unis** | **[Zyph](https://osu.ppy.sh/users/1600432)**, [EtienneXC](https://osu.ppy.sh/users/5610085), [TheToaphster](https://osu.ppy.sh/users/7616811), [Chrubble](https://osu.ppy.sh/users/2594280), [stupud man](https://osu.ppy.sh/users/2141612), [xElectroGH](https://osu.ppy.sh/users/3251373) |
| ::{ flag=VE }:: | **Venezuela** | **[Cyssel](https://osu.ppy.sh/users/6394478)**, [Satoruu](https://osu.ppy.sh/users/2243452), [Shinsekai-](https://osu.ppy.sh/users/2140739), [Marunne](https://osu.ppy.sh/users/8841450), [Prancer](https://osu.ppy.sh/users/1821350), [\[-Gearfrik-\]](https://osu.ppy.sh/users/8198818) |

## Groupes

| Groupe | Top seed | High seed | Low seed | Unseeded |
| :-: | :-- | :-- | :-- | :-- |

> | **A** | ::{ flag=BR }:: Brésil | ::{ flag=PL }:: Polognee | ::{ flag=PH }:: Philippines | ::{ flag=VE }:: Venezuela |

Reviewer: ```suggestion
| **A** | ::{ flag=BR }:: Brésil | ::{ flag=PL }:: Pologne | ::{ flag=PH }:: Philippines | ::{ flag=VE }:: Venezuela |
```

| **B** | ::{ flag=GB }:: Royaume-Uni | ::{ flag=MY }:: Malaisie | ::{ flag=HK }:: Hong Kong | ::{ flag=NO }:: Norvège |

> | **C** | ::{ flag=CN }:: Chine | ::{ flag=FR }:: France | ::{ flag=AU }:: Australie | ::{ flag=TW }:: Taiwan |

Reviewer: ```suggestion
| **C** | ::{ flag=CN }:: Chine | ::{ flag=FR }:: France | ::{ flag=AU }:: Australie | ::{ flag=TW }:: Taïwan |
```

| **D** | ::{ flag=US }:: États-Unis | ::{ flag=AR }:: Argentine | ::{ flag=MX }:: Mexique | ::{ flag=DE }:: Allemagne |

> | **E** | ::{ flag=KR }:: Corée du Sud | ::{ flag=SE }:: Suède | ::{ flag=RU }:: Russian Federation | ::{ flag=FI }:: Finland |

Reviewer: ```suggestion
| **E** | ::{ flag=KR }:: Corée du Sud | ::{ flag=SE }:: Suède | ::{ flag=RU }:: Fédération de Russie | ::{ flag=FI }:: Finlande |
```

> | **F** | ::{ flag=ID }:: Indonesie | ::{ flag=IT }:: Italie | ::{ flag=ES }:: Espagne | ::{ flag=NZ }:: Nouvelle-Zélande |

Reviewer: ```suggestion
| **F** | ::{ flag=ID }:: Indonésie | ::{ flag=IT }:: Italie | ::{ flag=ES }:: Espagne | ::{ flag=NZ }:: Nouvelle-Zélande |
```

| **G** | ::{ flag=JP }:: Japon | ::{ flag=CA }:: Canada | ::{ flag=CH }:: Suisse | ::{ flag=NL }:: Pays-Bas |
| **H** | ::{ flag=CL }:: Chili | ::{ flag=TH }:: Thaïlande | ::{ flag=SG }:: Singapour | ::{ flag=BE }:: Belgique |

> ![](img/bracket.jpg "MWC 4K 2017 bracket")

Reviewer: ```suggestion
![](img/bracket.jpg "Bracket de la MWC 4K 2017")
```

## Podium

> ![](img/podium.jpg "MWC 4K 2017 podium")

Reviewer: ```suggestion
![](img/podium.jpg "Podium de la MWC 4K 2017")
```

## Résultats des matchs

### Finales - Semaine 2

Samedi, 16 Septembre 2017:

| Équipe 1 |  |  | Équipe 2 | Lien du match |
| --: | :-: | :-: | :-- | :-- |
| États-Unis ::{ flag=US }:: | 0 | **7** | ::{ flag=KR }:: **Corée du Sud** | [#1](https://osu.ppy.sh/community/matches/36111799) |

### Finales - Semaine 1

> Samedi, 9 Septembre 2017:

Reviewer: ```suggestion
Samedi 9 septembre 2017 :
```

| Équipe 1 |  |  | Équipe 2 | Lien du match |
| --: | :-: | :-: | :-- | :-- |
| **Japon** ::{ flag=JP }:: | **6** | 0 | ::{ flag=CL }:: Chili | [#1](https://osu.ppy.sh/community/matches/35952218) |
| Royaume-Uni ::{ flag=GB }:: | 0 | **6** | ::{ flag=BR }:: **Brésil** | [#1](https://osu.ppy.sh/community/matches/35955973) |

> Dimanche, 10 Septembre 2017:

Reviewer: ```suggestion
Dimanche 10 septembre 2017 :
```

| Équipe 1 |  |  | Équipe 2 | Lien du match |
| --: | :-: | :-: | :-- | :-- |
| **Corée du Sud** ::{ flag=KR }:: | **6** | 0 | ::{ flag=US }:: États-Unis | [#1](https://osu.ppy.sh/community/matches/35972975) |
| **Brésil** ::{ flag=BR }:: | **6** | 1 | ::{ flag=JP }:: Japon | [#1](https://osu.ppy.sh/community/matches/35974418) |
| Brésil ::{ flag=BR }:: | 5 | **6** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35998794) |

### Demi-finale

> Samedi, 2 Septembre 2017:

Reviewer: ```suggestion
Samedi 2 septembre 2017 :
```

| Équipe 1 |  |  | Équipe 2 | Lien du match |
| --: | :-: | :-: | :-- | :-- |
| Suède ::{ flag=SE }:: | 1 | **6** | ::{ flag=JP }:: **Japon** | [#1](https://osu.ppy.sh/community/matches/35769761) |
| **Chine** ::{ flag=CN }:: | **6** | 2 | ::{ flag=FR }:: France | [#1](https://osu.ppy.sh/community/matches/35771726) |
| Brésil ::{ flag=BR }:: | 0 | **6** | ::{ flag=KR }:: **Corée du Sud** | [#1](https://osu.ppy.sh/community/matches/35774130) |
| Pologne ::{ flag=PL }:: | 1 | **6** | ::{ flag=ID }:: **Indonésie** | [#1](https://osu.ppy.sh/community/matches/35777392) |
| **Royaume-Uni** ::{ flag=GB }:: | **6** | 3 | ::{ flag=CA }:: Canada | [#1](https://osu.ppy.sh/community/matches/35779902) |
| **États-Unis** ::{ flag=US }:: | **6** | 0 | ::{ flag=CL }:: Chili | [#1](https://osu.ppy.sh/community/matches/35783065) |

> Dimanche, 3 Septembre 2017:

Reviewer: ```suggestion
Dimanche 3 septembre 2017 :
```

| Équipe 1 |  |  | Équipe 2 | Lien du match |
| --: | :-: | :-: | :-- | :-- |
| **Japon** ::{ flag=JP }:: | **6** | 3 | ::{ flag=ID }:: Indonésie | [#1](https://osu.ppy.sh/community/matches/35804308) |
| Chine ::{ flag=CN }:: | 2 | **6** | ::{ flag=GB }:: **Royaume-Uni** | [#1](https://osu.ppy.sh/community/matches/35807134) |

### Quarts de finale

> Samedi, 26 Août 2017:

Reviewer: ```suggestion
Samedi 26 août 2017 :
```

| Équipe 1 |  |  | Équipe 2 | Lien du match |
| --: | :-: | :-: | :-- | :-: |
| France ::{ flag=FR }:: | 4 | **5** | ::{ flag=CL }:: **Chili** | [#1](https://osu.ppy.sh/community/matches/35585664) |
| Argentine ::{ flag=AR }:: | 1 | **5** | ::{ flag=GB }:: **Royaume-Uni** | [#1](https://osu.ppy.sh/community/matches/35588367) |
| Canada ::{ flag=CA }:: | 0 | **5** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35590366) |

> Dimanche, 27 Août 2017:

Reviewer: ```suggestion
Dimanche 27 août 2017 :
```

| Équipe 1 |  |  | Équipe 2 | Lien du match |
| --: | :-: | :-: | :-- | :-: |
| **Corée du Sud** ::{ flag=KR }:: | **5** | 0 | ::{ flag=JP }:: Japon | [#1](https://osu.ppy.sh/community/matches/35601021) |
| Nouvelle-Zélande ::{ flag=NZ }:: | 0 | **5** | ::{ flag=PL }:: **Pologne** | [#1](https://osu.ppy.sh/community/matches/35602514) |
| Singapour ::{ flag=SG }:: | 1 | **5** | ::{ flag=CN }:: **Chine** | [#1](https://osu.ppy.sh/community/matches/35604433) |
| Hong-Kong ::{ flag=HK }:: | 0 | **5** | ::{ flag=SE }:: **Suède** | *victoire par défaut* |
| **Brésil** ::{ flag=BR }:: | **5** | 1 | ::{ flag=ID }:: Indonésie | [#1](https://osu.ppy.sh/community/matches/35608732) |

### Huitièmes de finale

> Dimanche, 20 Août 2017:

Reviewer: ```suggestion
Dimanche 20 août 2017 :
```

| Équipe 1 |  |  | Équipe 2 | Lien du match |
| --: | :-: | :-: | :-- | :-: |
| Nouvelle-Zélande ::{ flag=NZ }:: | 1 | **5** | ::{ flag=FR }:: **France** | [#1](https://osu.ppy.sh/community/matches/35418105) |
| **Indonésie** ::{ flag=ID }:: | **5** | 1 | ::{ flag=CN }:: Chine | [#1](https://osu.ppy.sh/community/matches/35419189) |
| Royaume-Uni ::{ flag=GB }:: | 3 | **5** | ::{ flag=JP }:: **Japon** | [#1](https://osu.ppy.sh/community/matches/35420314) |
| **Corée du Sud** ::{ flag=KR }:: | **5** | 0 | ::{ flag=AR }:: Argentine | [#1](https://osu.ppy.sh/community/matches/35421542) |
| **Brésil** ::{ flag=BR }:: | **5** | 0 | ::{ flag=SG }:: Singapour | [#1](https://osu.ppy.sh/community/matches/35423100) |
| Hong-Kong ::{ flag=HK }:: | 4 | **5** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/35424487) |
| Pologne ::{ flag=PL }:: | 0 | **5** | ::{ flag=CL }:: **Chili** | [#1](https://osu.ppy.sh/community/matches/35429710) |
| Suède ::{ flag=SE }:: | 0 | **5** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35431890) |

### Phases de poules

> Samedi, 12 Août 2017:

Reviewer: ```suggestion
Samedi 12 août 2017 :
```

| Belgique ::{ flag=BE }:: | 0 | **4** | ::{ flag=CL }:: **Chili** | [#1](https://osu.ppy.sh/community/matches/35214432) |
| Pologne ::{ flag=PL }:: | 0 | **4** | ::{ flag=BR }:: **Brésil** | [#1](https://osu.ppy.sh/community/matches/35218496) |
| Allemagne ::{ flag=DE }:: | 0 | **4** | ::{ flag=MX }:: **Mexique** | [#1](https://osu.ppy.sh/community/matches/35219133) |
| Argentine ::{ flag=AR }:: | 1 | **4** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35218503) |
| Pays-Bas ::{ flag=NL }:: | 1 | **4** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/35218508) |
| Venezuela ::{ flag=VE }:: | 0 | **4** | ::{ flag=PL }:: **Pologne** | [#1](https://osu.ppy.sh/community/matches/35219912) |
| Allemagne ::{ flag=DE }:: | 0 | **4** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35219931) |
| Mexique ::{ flag=MX }:: | 0 | **4** | ::{ flag=AR }:: **Argentine** | [#1](https://osu.ppy.sh/community/matches/35219934) |
| Suisse ::{ flag=CH }:: | 0 | **4** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/35219940) |

> Dimanche, 13 Août 2017:

Reviewer: ```suggestion
Dimanche 13 août 2017 :
```
